src/components/AddComment.jsx

```diff
@@ -0,0 +1,15 @@
+const AddComment = () => {
+    return (
+        <div className="AddComment">
```

Comment:
Is there a more semantic html tag you could use to signify this is an area for user input?

Reply:
changed to form tag